tensorflow/compiler/xla/service/gpu/matmul_utils.h

```diff
@@ -120,7 +120,8 @@ struct GemmConfig {
 };
 
 StatusOr<se::blas::ComputationType> GetBlasComputationType(
-    PrimitiveType dtype, int64_t compute_precision);
+    PrimitiveType dtype, int64_t compute_precision,
+    PrimitiveType lhs_dtype = PRIMITIVE_TYPE_INVALID);
```

Comment:
I would not give a default value here, but instead modifier the three callers (two in matmul_utils.cc, one in gemm_rewriter.cc) to pass the lhs_dtype.